public/app/features/canvas/elements/windTurbine.tsx

```diff
@@ -79,6 +79,11 @@ export const windTurbineItem: CanvasElementItem<any, any> = {
 
   getNewOptions: (options) => ({
     ...options,
+    background: {
+      color: {
+        fixed: 'transparent',
+      },
+    },
```

Comment:
If possible it might be nice to update the default size of windTurbine to better match its shape (width 100, height 150?)
![Screen Shot 2022-07-29 at 12 18 44 PM](https://user-images.githubusercontent.com/22381771/181829704-65bef2bd-4a1d-4326-98fe-d46ebd648e48.png)